hugolib/page.go

```diff
@@ -604,9 +645,10 @@ func (s *Site) newPage(filename string) *Page {
 		Site:         &s.Info,
 		s:            s,
 	}
+	p.layoutIdentifier = pageLayoutIdentifier{p}
```

Comment:
Please wait with the code comments.

On this one, the reason is to make the values unexported (i.e. not visible to the templates).